File: ts-sdk/src/sdk/models/shared/listscopesresponse.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import { Scope } from "./scope";


export class ListScopesResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=data", elemType: Scope })
  data?: Scope[];
}

File: python-sdk/src/formanceapi/models/shared/listscopesresponse.py
import dataclasses
from ..shared import scope as shared_scope
from dataclasses_json import dataclass_json
from formanceapi import utils
from typing import Optional


@dataclass_json
@dataclasses.dataclass
class ListScopesResponse:
    data: Optional[list[shared_scope.Scope]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('data') }})
    